CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @ndowmon
```

Comment:
This means that you couldn't create a PR and approve a PR! :D 

`@jupiterone/integrations`? 

Additionally, we should add the security team like this:

```
* @jupiterone/integrations
CODEOWNERS @jupiterone/security
```

The security team is supposed to approve any changes to the `CODEOWNERS` file.